corehq/blobs/migrate.py

```diff
@@ -422,6 +422,7 @@ def get_document_provider(self, date_range=None):
         iteration_key = self.iteration_key
         if date_range:
             (start, end) = date_range
+            self.reindexer.date_range = date_range
```

Comment:
Why was this necessary? It is already assigned above:

https://github.com/dimagi/commcare-hq/blob/4f44d15891fb9a915e8d541d3f0b15c385792fb3/corehq/blobs/migrate.py#L417

Would be best to only assign in one place. Maybe that one should be removed?

Reply:
Yeah, I was worried if removing that would break something else. But looks like it shouldn't, will update in a separate PR.